src/Save.java

```diff
@@ -0,0 +1,35 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+public class Save {
```

Comment:
I'd recommend passing a MainWindow parameter to gain access to GameWindow variables.